packages/forms/resources/views/components/tags-input.blade.php

```diff
@@ -10,10 +10,12 @@
     <div
         ax-load
         ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('tags-input', 'filament/forms') }}"
-        x-data="tagsInputFormComponent({
-                    state: $wire.{{ $applyStateBindingModifiers("\$entangle('{$statePath}')") }},
-                    splitKeys: @js($splitKeys),
-                })"
+        x-init="function() {
```

Comment:
This can be refactored to `() => {}` right?

Reply:
I think even `() => (tagsInputFormComponent(...))` which replaces the `return` IIRC

Reply:
Oh yeah, I saw multiple lines while it's just a single function call lol.